internal/innosat/source.go

```diff
@@ -34,7 +48,7 @@ func (sph *SourcePacketHeader) Version() uint {
 }
 
 // Type is either Telecommand or Telemetry
-func (sph *SourcePacketHeader) Type() SourcePacketHeaderType {
+func (sph SourcePacketHeader) Type() SourcePacketHeaderType {
```

Comment:
why take a copy of the struct? Or am I missing something about the point of removing the `*`?